src/agent/coverage/src/code.rs

```diff
@@ -195,6 +195,20 @@ impl ModuleIndex {
             symbols,
         })
     }
+
+    #[cfg(target_os = "windows")]
+    pub fn index_pe(path: ModulePath, pe: &goblin::pe::PE) -> Self {
```

Comment:
With no impl or comments, it's hard to tell why there is a `parse_elf` for Linux but no `index_elf`, and a `index_pe` for Windows bug no `parse_pe`.

Reply:
Will clean up the naming and document.